src/windows/track_list.c

```diff
@@ -181,7 +181,10 @@ static void window_track_list_close(rct_window *w)
 static void window_track_list_select(rct_window *w, int index)
 {
 	w->track_list.var_480 = index;
-
+	if (_loadedTrackDesign == NULL)
+		window_error_open(STR_CANT_BUILD_PARK_ENTRANCE_HERE, STR_TRACK_LOAD_FAILED_ERROR);
+	else
```

Comment:
Would it be better to have ''return;'' would that display and work properly instead of this? As it feels like I should move this code further down or something xox